py-polars/src/lazy/dataframe.rs

```diff
@@ -592,6 +605,7 @@ impl PyLazyFrame {
 
     #[allow(clippy::too_many_arguments)]
     #[cfg(feature = "asof_join")]
+    #[pyo3(signature = (other, left_on, right_on, left_by, right_by, allow_parallel, force_parallel, suffix, strategy, tolerance, tolerance_str))]
```

Comment:
Must such long macro lines be reformatted?

Reply:
If cargo fmt doesn't do it, it seems fine. :+1: